.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 9.x, 10.x, 11.x, 12.x, 13.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm test
+      env:
+        CI: true
```

Comment:
What does `CI: true` do?

Reply:
It's in the boilerplate example (https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-nodejs)    I believe it sets the env variable CI to true.  This could be useful in the future however I can remove it if you want

Reply:
Ah ok. That makes sense. It's OK to leave it in. It may be useful at some point for the test suite to know it is running in CI.